tests/test_worktree_management.py

```
@@ -191,3 +191,128 @@ def test_list_worktrees_empty_when_none_exist(tmp_path, git_env):
+    subprocess.run(
+        ["git", "init", "--bare", str(bare_path)],
+        check=True,
+        capture_output=True,
+        text=True,
+    )
+
+
+def _init_repo_with_remote(repo: Path, remote_path: Path) -> None:
+    """Initialize a repo and add a local bare repo as 'origin'."""
+    _init_repo(repo)
+    subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)